src/scripts/ci/travis/build.sh

```diff
@@ -13,7 +13,8 @@ elif [ "$BUILD_MODE" = "sanitizer" ]; then
 fi
 
 if [ "$MODULES" = "min" ]; then
-    CFG_FLAGS+=(--minimized-build --enable-modules=base)
+    # Added chacha for passing the minimum modules compilation
+    CFG_FLAGS+=(--minimized-build --enable-modules=base,chacha)
```

Comment:
This change should not be required. Code in the tests should check if BOTAN_HAS_CHACHA is defined.